setup.cfg

```diff
@@ -59,3 +59,4 @@ junit_family = xunit2
 filterwarnings =
     error
     ignore:numpy.ndarray size changed:RuntimeWarning
+addopts = "-p no:legacypth"
```

Comment:
typo?
```suggestion
addopts = "-p no:legacypath"
```

Reply:
Oops! Thanks for the catch. Fixed in https://github.com/spacetelescope/ci_watson/pull/64/commits/c0e852b8603a118e2ae56057a6f8243ff15f570d